Sources/CvInfos.h

```diff
@@ -232,22 +232,22 @@ class CvDiplomacyResponse
 	CvDiplomacyResponse();
 	virtual ~CvDiplomacyResponse();
 
-	int getNumDiplomacyText();
+	int getNumDiplomacyText() const;
 	void setNumDiplomacyText(int i);
 
-	bool getCivilizationTypes(int i);
+	bool getCivilizationTypes(const int i) const;
```

Comment:
Its not really idiomatic to make integral type parameters const (int, float, char), only references and pointers. It isn't wrong per se, but you will have an uphill battle against the rest of the C++ community to get them to do it 🙂